Dockerfile

```diff
@@ -223,30 +86,31 @@ WORKDIR /artifacts
 
 # Install EnergyPlus
 RUN --mount=type=bind,from=energyplus-dependencies,source=/artifacts,target=/artifacts set -eux; \
-  mkdir ${ENERGYPLUS_DIR}; \
+  mkdir -p ${ENERGYPLUS_DIR}; \
   tar -C $ENERGYPLUS_DIR/ --strip-components=1 -xzf energyplus.tar.gz; \
   cd ${ENERGYPLUS_DIR}; \
+  cp -n -r ${ENERGYPLUS_DIR}/python_lib/* /usr/local/lib/python3.12; \
```

Comment:
This copies the energyplus python standard library into the same folder as the system's runtime. `-n` prevents the system's runtime from being replaced by files coming from `python_lib`. After this `python_lib` is deleted and linked to `/usr/local/lib/python3.12`. This helps trim down the image size by reducing duplicate information, it also makes sure e+ and the system are using the same python.